pkg/ztunnel/iptables/inpod.go

```diff
@@ -0,0 +1,480 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package iptables
+
+import (
+	"errors"
+	"fmt"
+	"log/slog"
+	"net"
+	"os"
+
+	"github.com/coreos/go-iptables/iptables"
+	"github.com/vishvananda/netlink"
+	"golang.org/x/sys/unix"
+
+	"github.com/cilium/cilium/pkg/datapath/linux/route"
+)
+
+const (
+	InpodPreroutingChain = "CILIUM_PREROUTING"
+	InpodOutputChain     = "CILIUM_OUTPUT"
+
+	RouteTableInbound = 100
+
+	InpodRulePriority = 32764
```

Comment:
I know this value is probably coming from looking at a ztunnel deployment, but would be good to call out magic numbers like this and 0x111, informing the reader that these are pulled in from a stock ztunnel deployment.